nukebuild/ApiDiffHelper.cs

```diff
@@ -311,7 +311,12 @@ public static async Task<GlobalDiffInfo> DownloadAndExtractPackagesAsync(
             // Download baseline package
             memoryStream.Position = 0L;
             memoryStream.SetLength(0L);
-            await DownloadBaselinePackageAsync(memoryStream, downloadContext, packageId, baselineVersion);
```

Comment:
Should be moved to another PR, but we should distinguish between a package not existing and one whose download failed.

Reply:
We can revert that change after the final API baseline has been generated